var/spack/repos/builtin/packages/py-scipy/package.py

```diff
@@ -32,9 +32,14 @@ class PyScipy(Package):
     version('0.17.0', '5ff2971e1ce90e762c59d2cd84837224')
     version('0.15.1', 'be56cd8e60591d6332aac792a5880110')
     version('0.15.0', '639112f077f0aeb6d80718dc5019dc7a')
+    
+    variant('atlas', default=False,  description='Compile with ATLAS')
```

Comment:
IMHO should not be a variant.
```
spack install py-scipy ^atlas
```
should use it for both `blas` and `lapack`. That's when 
`if 'atlas' in spec:` will be triggered.